app/assets/stylesheets/header.css.scss

```diff
@@ -0,0 +1,116 @@
+header {
+  min-height: 50px;
+}
+
+h1 {
+  color: $dark-blue;
+  font-size: 3em;
+  font-weight: bold;
+  text-align: center;
+}
+
+h2 {
+  font-size: 2em;
+  font-style: italic;
+  font-weight: bold;
+  color: $dark-blue;
+}
+
+.navigation-bar {
+  background: transparentize($blue, 0.2);
+  border-style: ridge none;
+  min-height: 50px;
+  position: relative;
+}
+
+#nav {
+  font-weight: bold;
+  left: 30%;
+  list-style-type: none;
+  margin-top: 0;
+  position: absolute;
+  width: 70%;
+
+  > li {
+    display: inline-block;
+    padding: 15px;
+  }
+
+  a {
+    text-decoration: none;
+   
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+}
+
+.dropdown {
+  background: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.